realtime/earthquake/processor/shake_event.py

```diff
@@ -1,1898 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-InaSAFE Disaster risk assessment tool developed by AusAid and World Bank
-- **Functionality related to shake events.**
-
-Contact : ole.moller.nielsen@gmail.com
-
-.. note:: This program is free software; you can redistribute it and/or modify
-     it under the terms of the GNU General Public License as published by
-     the Free Software Foundation; either version 2 of the License, or
-     (at your option) any later version.
-
-"""
-__author__ = 'tim@kartoza.com'
```

Comment:
Accidentally duplicated file, not needed.
The real one is in:
realtime/earthquake/shake_event.py